### PR TITLE
feat: add more logs to wandb

### PR DESCRIPTION
**Pull request by karootplx**

### CODE REVIEW

File: commons/orm.py [hunk 1]
```diff
@@ -521,3 +521,46 @@ async def get_validator_score() -> torch.Tensor | None:
...
+            dojo_task_id (str): The Dojo task ID to search for.
+
+        Returns:
+            dict[str, Optional[float]]: A dictionary mapping model ID to score (which may be None) for the given Dojo task ID.
+        """
+        try:
+            # First, find the Feedback_Request_Model with the given dojo_task_id
+            feedback_request = await Feedback_Request_Model.prisma().find_first(
+                where=Feedback_Request_ModelWhereInput(dojo_task_id=dojo_task_id),
+                include={"completions": True},
```
Comment (jarvis8x7b) [inline commons/orm.py]: 1. add `child requests: {"some": {}}` as a filter to get the validator's record so u can also grab ground truth, these do not exist on the miner records i believe.
2. add `parent_id = <id of record from step 1>` to get the miner responses for that record

File: neurons/validator.py [hunk 2]
```diff
@@ -214,16 +214,35 @@ async def log_wandb():
                                 f"📝 Mean miner scores across different criteria: consensus shape:{mean_weighted_consensus_scores}, gt shape:{mean_weighted_gt_scores}"
                             )
 
+                            ground_truth_scores = [
+                                miner_scores.ground_truth.score
+                                for miner_scores in criteria_to_miner_score.values()
+                            ]
```
Comment (jarvis8x7b) [inline neurons/validator.py]: sorry for the confusing field names, this is actually the scores that each miner achieved when comparing their outputs to the synthetic ground truth and not the raw ground truth

File: commons/orm.py (hunk 1 above)
Comment (karootplx) [reply commons/orm.py]: did this by grabbing parent request id

File: neurons/validator.py (hunk 2 above)
Comment (karootplx) [reply neurons/validator.py]: oops mb. removed

**Review by jarvis8x7b — APPROVED** [review]
lgtm

**Pull request merged**